client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/TransferInvestigationsDialogs/TransferInvestigationDialogNote.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { Typography } from '@material-ui/core';
```

Comment:
should be in the first section with react

Reply:
Fixed